uvicorn/logging.py

```diff
@@ -96,7 +103,7 @@ def formatMessage(self, record):
             http_version,
             status_code,
         ) = recordcopy.args
-        status_code = self.get_status_code(status_code)
+        status_code = self.get_status_code(status_code)  # type: ignore
```

Comment:
If the status code is coming in from the log record as a string, like `418`, it could be cast to an `int`.

`status_code = self.get_status_code(int(status_code))`

This would work well with `http.HTTPStatus(status_code).phrase`.

```py
>>> import http
>>> status_code = int("418")
>>> http.HTTPStatus(status_code).phrase
"I'm a Teapot"
```

Reply:
Thanks @br3ndonland